go.mod

```diff
@@ -27,9 +29,12 @@ require (
 	google.golang.org/api v0.40.0
 	google.golang.org/genproto v0.0.0-20210226172003-ab064af71705
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
-	k8s.io/client-go v0.20.4
+	k8s.io/apimachinery v0.20.5
+	k8s.io/client-go v11.0.0+incompatible
 )
 
+replace k8s.io/client-go => k8s.io/client-go v0.20.5
```

Comment:
```suggestion
```

Reply:
Sure @Issif

Reply:
Hey @Issif, I tried these changes locally, and `go mod tidy` and `go build` gave errors, I believe that these are necessary. The adapter requires the old client-go unfortunately.